curriculum/Chapter-?-Databases.md

```diff
@@ -0,0 +1,30 @@
+# Chapter ?: Databases
+
+## Objectives
+
+After completing this chapter, you will be able to:
+
+- Understand the role and importance of databases in application development.
+- Differentiate between graph, document, and relational-based databases.
```

Comment:
- Differentiate between graph, document, and relational databases.

Reply:
Consider adding an assets folder, and providing illustrations to visualize their uniqueness.